Client/Blogger/src/App.jsx

```diff
@@ -0,0 +1,43 @@
+import { BrowserRouter, Routes, Route } from "react-router-dom";
+import "bootstrap/dist/css/bootstrap.min.css";
+import { ToastContainer, toast } from "react-toastify";
+import { Container, Nav } from "react-bootstrap";
+import HomePage from "./pages/Home";
+import BlogPage from "./pages/BlogPage";
+import NavBar from "./Components/NavBar";
+import FormPage from "./pages/FormPage";
+import { Provider } from "react-redux";
+import reduxStore from "./Redux/store";
+import CreateBlog from "./pages/CreateBlog";
+import EditBlogPage from "./pages/EditBlogPage";
+import "react-toastify/dist/ReactToastify.css";
+
+
+import "./Styles/app.css";
+import Footer from "./Components/Footer";
+import MediaPage from "./pages/Media";
+function App() {
+  return (
+    <Provider store={reduxStore}>
+      <ToastContainer />
+      <BrowserRouter>
+        <Container>
+          <NavBar />
+          <Routes>
+            <Route path="/" element={<HomePage />} />
+            <Route path="/auth" element={<FormPage />} />
+            <Route path="/media" element={<MediaPage />} />
+            <Route path="/blog">
```

Comment:
Consider moving the route paths to a constants file and use those constants wherever you're using the paths. By doing so, if any changes are required, you can update them in a single place. This approach applies not only to route paths but also to any repetitive strings.